src/Mod/BIM/ArchWindow.py

```diff
@@ -514,12 +514,25 @@ def appLinkExecute(self, obj, linkObj, index, linkElement):
         # Execute features in the SketchArch External Add-on
         self.executeSketchArchFeatures(obj, linkObj)
 
-    def getSubVolume(self,obj,plac=None):
+    def getSubFace():
+        "returns a subface for creation of subvolume for cutting in a base object"
+        # creation of subface from HoleWire (getSubWire)
+        pass
+
```

Comment:
This looks to be both incomplete and unused. Maybe it should be removed?

Reply:
Indeed this is incomplete, just some ground works for more robust approach in TODO list.

Reply:
Ok, if you want to keep it, how about:
1. add self as an argument
2. raise an exception if someone would call it.
```suggestion
    def getSubFace(self):
        "returns a subface for creation of subvolume for cutting in a base object"
        # creation of subface from HoleWire (getSubWire)
        raise NotImplementedError
```

Reply:
Done, thanks